### ISSUE TITLE
Cells with `run` to execute Python scripts shouldn't be formatted

### ISSUE THREAD

**Issue by nobodyinperson** [ryantam626/jupyterlab_code_formatter#236]
**Checklist prior to opening an issue**
- [x] I have followed fully the installation steps laid out in the documentation site.
- [x] I have restarted jupyterlab.
- [x] I have read the FAQ section in the documentation site.

**Describe the bug**

Cells starting with `run` to execute a Python script yield a formatting error.

**Diagnostic commands**
Please attach the output of the following commands (please format them properly)

<details>
<summary>`pip freeze`</summary>

```
aiofiles @ file:///build/python-aiofiles/src/aiofiles/dist/aiofiles-0.8.0-py3-none-any.whl
aiohttp==3.8.1
aiohttp-socks==0.7.1
aiorpcX==0.18.7
aiosignal==1.2.0
ajsonrpc==1.2.0
anyio==3.5.0
apipkg==2.1.0
apparmor==3.0.3
appdirs==1.4.4
application-utility==1.3.2
argon2-cffi==21.1.0
asciinema==2.0.2
asgiref==3.5.0
asttokens==2.0.5
async-generator==1.10
async-timeout==4.0.2
attrs==21.4.0
Babel==2.9.1
backcall==0.2.0
beautifulsoup4==4.10.0
bitstring==3.1.9
black==22.1.0
bleach==4.1.0
bottle==0.12.19
btrfsutil==5.16
CacheControl==0.12.6
catfish==4.16.3
ceph==1.0.0
ceph-volume==1.0.0
cephfs==2.0.0
cephfs-shell==0.0.1
certifi==2021.10.8
cffi==1.15.0
cftime==1.5.1
chardet==4.0.0
charset-normalizer==2.0.10
click==8.0.3
cloudpickle==2.0.0
colorama==0.4.4
contextlib2==0.6.0.post1
cram==0.7
cryptography==36.0.1
cupshelpers==1.0
cycler==0.10.0
debugpy==1.5.1
decorator==5.1.1
defusedxml==0.7.1
deprecation==2.1.0
distlib==0.3.4
distro==1.6.0
dnspython==2.2.0
docopt==0.6.2
docutils==0.16
ecdsa==0.17.0
Electrum==4.1.5
entrypoints==0.4
executing==0.8.2
fonttools==4.29.1
frozenlist==1.2.0
future==0.18.2
GDAL==3.4.0
gitdb==4.0.7
GitPython==3.1.24
greenlet==1.1.2
gufw==21.4.0
h11==0.12.0
html5lib==1.1
httptools==0.3.0
idna==3.3
ifaddr==0.1.7
importlib-metadata==4.8.1
inflect==5.4.0
iniconfig==1.1.1
ipykernel==6.8.0
ipympl==0.8.7
ipython==8.0.1
ipython-genutils==0.2.0
ipywidgets==7.6.5
isc==2.0
isort==5.10.1
jedi==0.18.1
Jinja2==3.0.3
json5==0.9.6
jsonrpclib-pelix==0.4.3.1
jsonschema==3.2.0
jupyter-client==7.1.2
jupyter-console==6.4.0
jupyter-core==4.9.1
jupyter-packaging==0.11.1
jupyter-server==1.13.4
jupyterlab==3.2.8
jupyterlab-code-formatter==1.4.10
jupyterlab-pygments==0.1.2
jupyterlab-server==2.10.3
jupyterlab-widgets==1.0.2
keyutils==0.6
kiwisolver==1.3.2
lensfun==0.3.95
LibAppArmor==3.0.3
LICOR840A @ file:///home/yann/code/tue-umphy/instruments/python3-LICOR840A
lightdm-gtk-greeter-settings==1.2.2
lit==13.0.0.dev0
louis==3.20.0
lxml==4.7.1
manjaro-sdk==0.1
Markdown==3.3.6
MarkupSafe==2.0.1
marshmallow==3.14.1
matplotlib==3.5.1
matplotlib-inline==0.1.3
meld==3.20.4
menulibre==2.2.3
mistune==0.8.4
more-itertools==8.10.0
msgpack==1.0.3
mugshot==0.4.3
multidict==5.2.0
mypy-extensions==0.4.3
namcap==3.2.10
nbclassic==0.3.5
nbclient==0.5.10
nbconvert==6.4.1
nbformat==5.1.3
nbstripout==0.5.0
nest-asyncio==1.5.4
netCDF4==1.5.7
netifaces==0.11.0
netsnmp-python==1.0a1
notebook==6.4.8
npyscreen==4.10.5
numpy==1.21.5
opentimestamps==0.4.1
opentimestamps-client==0.7.0
ordered-set==4.0.2
OWSLib==0.24.1
packaging==20.9
pacman-mirrors==4.23.2
pandas==1.4.0
pandocfilters==1.5.0
parmesan==0.3.0+24.g7a4805b
parso==0.8.2
pathspec==0.9.0
patsy==0.5.2
pbkdf2==1.3
pdfminer==20191125
pep517==0.12.0
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.0.1
Pint==0.18
platformdirs==2.4.1
platformio==5.2.4
pluggy==0.13.1
ply==3.11
polt @ file:///home/yann/code/python3-polt
progress==1.6
prometheus-client==0.12.0
prompt-toolkit==3.0.26
protobuf==3.19.4
psutil==5.9.0
psycopg2==2.9.3
ptyprocess==0.7.0
pure-eval==0.2.2
py==1.11.0
pyaes==1.6.1
pyalpm==0.10.6
pybind11==2.9.0
pycairo==1.20.1
pycodestyle==2.8.0
pycparser==2.21
pycryptodomex==3.10.1
pycups==2.0.1
pycurl==7.44.1
pyelftools==0.27
Pygments==2.11.2
PyGObject==3.42.0
pynvim==0.4.3
pyOpenSSL==21.0.0
pyparsing==2.4.7
pyproj==3.0.1
PyQt5==5.15.6
PyQt5-sip==12.9.1
pyrsistent==0.18.1
pyserial==3.5
pysha3==1.0.2
PySide6==6.2.3
pysmbc==1.0.23
PySocks==1.7.1
pytest==6.2.5
pytest-cram==0.2.2
python-bitcoinlib==0.11.0
python-dateutil==2.8.2
python-distutils-extra==2.39
python-socks==2.0.3
pytz==2021.3
pyxdg==0.27
PyYAML==5.4.1
pyzmq==22.3.0
qrcode==7.3.1
QScintilla==2.13.1
rados==2.0.0
rbd==2.0.0
regex==2021.11.10
reportlab==3.6.6
requests==2.26.0
requests-unixsocket==0.3.0
resolvelib==0.5.5
retrying==1.3.3
rgw==2.0.0
RHCAL==0.1
rpm==4.17.0
scipy==1.7.3
SCons==4.3.0
semantic-version==2.8.5
Send2Trash==1.8.0
shiboken6==6.2.3
six==1.16.0
smmap==5.0.0
sniffio==1.2.0
soupsieve==2.2.1
spyder-kernels==3.0.0.dev0
stack-data==0.1.4
starlette==0.14.2
statsmodels==0.13.1
tabulate==0.8.9
TBB==0.2
team==1.0
terminado==0.13.1
testpath==0.5.0
thunar-plugins==0.3.2
TLPUI==1.5.0
toml==0.10.2
tomli==2.0.0
tomlkit==0.8.0
tornado==6.1
tqdm==4.62.3
traitlets==5.1.1
typing-extensions==4.0.1
udiskie==2.4.1
ufw==0.36.1
uncertainties==3.1.6
urllib3==1.26.8
uvicorn==0.17.0.post1
uvloop==0.16.0
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==0.59.0
websockets==10.1
widgetsnbextension==3.5.2
wsproto==1.0.0
wurlitzer==3.0.2
wxPython==4.0.7.post2
xarray==0.20.1
xdgspec==0.2.1
yarl==1.7.2
youtube-dl==2021.12.17
zeroconf==0.38.1
zipp==3.7.0
```
</details>

<details>
<summary>`jupyter labextension list`</summary>

```
JupyterLab v3.2.8
/usr/share/jupyter/labextensions
        jupyter-matplotlib v0.10.5 enabled OK
        @jupyter-widgets/jupyterlab-manager v3.0.1 enabled OK
        @ryantam626/jupyterlab_code_formatter v1.4.10 enabled OK

Other labextensions (built into JupyterLab)
   app dir: /usr/share/jupyter/lab

```
</details>

<details>
<summary>`jupyter serverextension list`</summary>

```
config dir: /etc/jupyter
    jupyterlab  enabled 
    - Validating...
      jupyterlab 3.2.8 OK
    jupyterlab_code_formatter  enabled 
    - Validating...
      jupyterlab_code_formatter 1.4.10 OK
```
</details>

**Screenshots**

![grafik](https://user-images.githubusercontent.com/19148271/153673273-611ee6ff-30e4-4fc3-a2f0-92e43844478a.png)

**Additional context**

similar issues: #218 #173 

Ideally, cells like this wouldn't be touched by the formatter. As a workaround, can one add a specific tag to a cell to prevent formatting?

**Comment by ryantam626** [comment]
Sorry for the late reply, was extremely burnt out and just switched off.

Thanks for the suggestion! I will have a think how to do this cleanly and do it in the coming days.

**Comment by ryantam626** [comment]
This should be now fixed in 1.4.11 - thanks again for flagging :pray:
